railties/lib/rails/generators/rails/app/templates/config/puma.rb.tt

```diff
@@ -35,7 +35,7 @@ plugin :tmp_restart
 
 <% unless skip_solid? -%>
 # Run the Solid Queue supervisor inside of Puma for single-server deployments
-plugin :solid_queue if ENV["SOLID_QUEUE_IN_PUMA"]
+plugin :solid_queue if ENV["SOLID_QUEUE_IN_PUMA"] == "true"
```

Comment:
The problem is that many people have the expectation that other values would work (like 1, yes, etc).

I rather keep the current behavior unless we already have a helper capable of converting strings to booleans (like active model/record does)